Plugins/Cuda/include/Acts/Plugins/Cuda/Utilities/CudaMatrix.cu

```diff
@@ -8,85 +8,95 @@
 
 #pragma once
 
+#include "Acts/Plugins/Cuda/Utilities/CpuMatrix.hpp"
```

Comment:
Are the includes not ordered in `.cu` files?

Reply:
I didn't configure anything, so no idea.